install.sh

```diff
@@ -23,13 +24,15 @@ esac
 
 deno_uri="https://github.com/denoland/deno/releases/download/${version}/deno_${os}_x64.gz"
 
-curl -fL# -o "$deno_dir/deno.gz" "$deno_uri"
-gunzip -df "$deno_dir/deno.gz"
-chmod +x "$deno_dir/deno"
+curl -fL# -o "$exe.gz" "$deno_uri"
+gunzip -df "$exe.gz"
+chmod +x "$exe"
 
-echo "Deno was installed successfully."
+echo "Deno was installed successfully to $exe"
 if command -v deno >/dev/null; then
   echo "Run 'deno --help' to get started."
 else
-  echo "Run '~/.deno/bin/deno --help' to get started."
+  echo "You must manually add it to your path:"
```

Comment:
Some people may choose to symlink to another directory already in the PATH. I think "you must add it to your PATH" sounds kinda weird. The "must" specifically makes you think "so if I don't add deno to the PATH, will it not work?".

Reply:
Noted.